_non-cosmos/ethereum/assetlist.json

```diff
@@ -1096,6 +1096,37 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/occamfi.png"
         }
       ]
+    },
+    {
+      "description": "The PAGE token is used for actions in the PageDAO NFT literary ecosystem and for DAO governance.",
+      "type_asset": "erc20",
+      "address": "0x60e683C6514Edd5F758A55b6f393BeBBAfaA8d5e",
+      "denom_units": [
+        {
+          "denom": "0x60e683C6514Edd5F758A55b6f393BeBBAfaA8d5e",
+          "exponent": 0,
+          "aliases": []
+        },
+        {
+          "denom": "PAGE",
+          "exponent": 8
+        }
+      ],
+      "base": "0x60e683C6514Edd5F758A55b6f393BeBBAfaA8d5e",
+      "name": "Page",
+      "display": "Page",
```

Comment:
This 'display' value is not defined under 'denom_units'. Usually the 'display' denom would be the ticker symbol but lower cased (e.g., 'page'), but it's also fine to use 'PAGE', which is what you've already defined for the exp:8 denom_unit.